packages/lib/src/components/Card/components/CardInput/types.ts

```diff
@@ -35,6 +37,7 @@ export interface CardInputState {
     additionalSelectElements: DualBrandSelectElement[];
     additionalSelectValue: string;
     billingAddress: object;
+    brand: string;
```

Comment:
```suggestion
    brand?: string;
```

Shouldn't this be optional @sponglord?